bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/AutomaticInboxProcessor.java

```diff
@@ -92,7 +92,8 @@ public void thingAdded(Inbox inbox, DiscoveryResult result) {
 
     private String getRepresentationValue(DiscoveryResult result) {
         return result.getRepresentationProperty() != null
-                ? (String) result.getProperties().get(result.getRepresentationProperty()) : null;
+                ? (String) result.getProperties().get(result.getRepresentationProperty())
```

Comment:
I would be interested what has changed on your IDE setup that causes this line break all of a sudden. Do you happen to know?